docs/mappings/gene_mapping_human.tsv

```diff
@@ -0,0 +1,2 @@
+subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification	mapping_date	author_id	subject_source	subject_source_version	object_source	object_source_version	confidence

```

Comment:
are all of these headings from sssom?

Reply:
Yes, there are up to 30 metadata elements we can include here. I just picked a standard set as an example.